From 371a2b62e6bbd53ca5efc1164553a4d334c936ee Mon Sep 17 00:00:00 2001
From: Michael Forney <mforney@mforney.org>
Date: Fri, 5 Jun 2020 01:17:51 -0700
Subject: [PATCH] Adjust style to prevent gcc warning

This is still implementation-defined, but matches the style used
elsewhere in this file and prevents a conversion overflow warning.
---
 bfd/elf.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/bfd/elf.c b/bfd/elf.c
index 9ca42e10d8..5ee61ffb24 100644
--- a/bfd/elf.c
+++ b/bfd/elf.c
@@ -733,8 +733,9 @@ setup_group (bfd *abfd, Elf_Internal_Shdr *hdr, asection *newsect)
 	      /* If all groups are invalid then fail.  */
 	      if (num_group == 0)
 		{
+		  num_group = (unsigned) -1;
 		  elf_tdata (abfd)->group_sect_ptr = NULL;
-		  elf_tdata (abfd)->num_group = num_group = -1;
+		  elf_tdata (abfd)->num_group = num_group;
 		  _bfd_error_handler
 		    (_("%pB: no valid group sections found"), abfd);
 		  bfd_set_error (bfd_error_bad_value);
-- 
2.28.0

